lib/common/constant.go

```diff
@@ -29,7 +29,14 @@ var (
 	// MaxTransactionsInBallot limits the maximum number of `Transaction`s in
 	// one proposed `Ballot`.
 	MaxTransactionsInBallot int = 1000
+
 	// MaxOperationsInTransaction limits the maximum number of `Operation`s in
 	// one `Transaction`.
 	MaxOperationsInTransaction int = 1000
+
+	// DefaultInflationRatio is the default inflation ratio. If the decimal
+	// points is over 17, the inflation amount will be 0, considering with
+	// `MaximumBalance`. The current value, `0.00000000000001` will increase
+	// `50BOS` in every block(current genesis balance is `5000000000000000`).
+	DefaultInflationRatio float64 = 0.00000000000001
 )
```

Comment:
I think DefaultInflationRatio should be 0.0000001.
Because current unit is GON.

Reply:
@soonkuk Yes, you right. I was confused the unit. Thanks.